commands/command_clean.go

```diff
@@ -97,8 +98,10 @@ func cleanCommand(cmd *cobra.Command, args []string) {
 		fileName = args[0]
 	}
 
-	if err := clean(os.Stdout, os.Stdin, fileName, -1); err != nil {
+	if nr, _, err := clean(os.Stdout, os.Stdin, fileName, -1); err != nil {
 		Error(err.Error())
+	} else if possiblyMalformedSmudge(nr) {
```

Comment:
I think this func name should be changed since this is not a smudge. What about something like `gitObjectTooBig()`?